docs/reference.md

```diff
@@ -750,7 +787,7 @@ This method returns a promise which is resolved or rejected when the authenticat
 completed. If successful, the promise will be fulfilled with an object containing authentication
 data about the signed-in user. If unsuccessful, the promise will be rejected with an `Error` object.
 
-Firebase currently supports Facebook, GitHub, Google, and Twitter authentication. Refer to
+Firebase currently supports Facebook, GitHub, Google, and Twitter authentication. Refer to the
```

Comment:
Firebase currently supports Facebook, GitHub, Google, and Twitter identity providers for use with $signInWithCredential. (We support other authentication methods: email/password, anonymous, and custom)